docs/sphinx/source/user_guide/forecasts.rst

```diff
@@ -4,6 +4,25 @@
 Forecasting
 ***********
 
+.. warning::
```

Comment:
I suppose this information could go in the whatsnew as well.  Open to suggestions both for its location and its content.

Reply:
what's new links to here, so I think it's fine.